### PR TITLE
Make nuget packages

### PR DESCRIPTION
**Pull request by dangershony**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      name: Checkout
+      
+    - name: Setup Node.js (12.x)
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+
+    - name: Variables
+      run: |
+        echo VERSION=$(npm run version --silent) >> $GITHUB_ENV
```
Comment (sondreb) [inline .github/workflows/build.yml]: This must be somehow modified to read the .props file and grab the version from there, as oppose to using node.js.

File: src/Lyn.Protocol/Lyn.Protocol.csproj [hunk 2]
```diff
@@ -4,6 +4,11 @@
     <TargetFramework>net5.0</TargetFramework>
     <Nullable>enable</Nullable>
     <RootNamespace>Lyn.Protocol</RootNamespace>
+    <Description>Library implementation of the lightning protocol.</Description>
+    <AssemblyTitle>Lyn.Protocol</AssemblyTitle>
+    <AssemblyName>Lyn.Protocol</AssemblyName>
```
Comment (sondreb) [inline src/Lyn.Protocol/Lyn.Protocol.csproj]: Not "Blockcore.Lyn.Protocol"?

File: src/Lyn.Types/Lyn.Types.csproj [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
-        <RootNamespace>Lyn.Types</RootNamespace>
-    </PropertyGroup>
-
-</Project>
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <RootNamespace>Lyn.Types</RootNamespace>
+    <PackageId>Lyn.Types</PackageId>
+    <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
```
Comment (sondreb) [inline src/Lyn.Types/Lyn.Types.csproj:8]: Shouldn't this package be deployed as NuGet? If so, remove this property.

**Review by sondreb — APPROVED** [review]
All the comments was fixed by myself or @dangershony .

File: .github/workflows/build.yml (hunk 1 above)
Comment (dangershony) [reply .github/workflows/build.yml]: But for now this will work right?

File: src/Lyn.Types/Lyn.Types.csproj (hunk 3 above)
Comment (dangershony) [reply src/Lyn.Types/Lyn.Types.csproj:8]: no it should not, just leave it false will be fine no?

**Pull request merged**